src/index.js

```diff
@@ -18,13 +18,15 @@ import { allDatasets } from "./datasetManifest";
 import { parseCSV } from "./csvReaderWrapper";
 import { parseJSON } from "./jsonReaderWrapper";
 import { TestDataLocations } from "./constants";
+import I18n from './i18n';
 
 export const store = createStore(rootReducer);
 
 let saveTrainedModel = null;
 let onContinue = null;
 
 export const initAll = function (options) {
+  I18n.initI18n(options.i18n);
```

Comment:
curious what `options.i18n` is/where it comes from

Reply:
And does the `initAll` call in `indexDev.js` need a reference to `i18n`?

Reply:
> curious what `options.i18n` is/where it comes from

This will be passed in `Ailab.js` in the code-dot-org project. All of our translations will live in the code-dot-org project and be passed along using `options.i18n`. You can see that happening here: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/ailab/Ailab.js#L206

> And does the initAll call in indexDev.js need a reference to i18n?
It doesn't need it because `indexDev.js` calls `index.js:initAll`. `index.js:initAll` doesn't need `i18n` provided because it will always default to English. It only needs i18n to be defined if we want to override the English with a different locale.
I will add this documentation.

Reply:
That makes sense! Thanks.